config/webpack.common.config.js

```diff
@@ -1,12 +1,14 @@
 const path = require('path');
 
+const BASENAME = process.env.BASENAME || '/';
```

Comment:
nit: would `PUBLIC_PATH` potentially be a more descriptive environment variable name here?

Reply:
In terms of webpack, that is probably more appropriate. But we also want to use this in react-router,  in [frontend-platform](https://github.com/edx/frontend-platform/pull/113). The configuration there is called `basename`. So, not sure if we should use one or the other.